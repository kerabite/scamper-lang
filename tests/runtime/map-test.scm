(define inc
  (lambda (x) (+ x 1)))

(define l (list 1 2 3 4 5 6 7 8 9 10))

(map inc l)

(map (lambda (p) (car p)) (list (cons "a" "b") (cons "c" "d") (cons "e" "f")))