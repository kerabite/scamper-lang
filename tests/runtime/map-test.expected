(list 2 3 4 5 6 7 8 9 10 11)
(list "a" "c" "e")
